CONTRIBUTING.rst

```diff
@@ -0,0 +1,124 @@
+..
+    Copyright (C) 2020 CERN.
+
+    Docker-Services-CLI is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/inveniosoftware/docker-services-cli/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "feature"
+is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+docker-services-cli could always use more documentation, whether as part of the
+official docker-services-cli docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at
+https://github.com/inveniosoftware/docker-services-cli/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `docker-services-cli` for local development.
+
+1. Fork the `inveniosoftware/docker-services-cli` repo on GitHub.
```

Comment:
We could directly add a link to do this: https://github.com/inveniosoftware/docker-services-cli/fork although I know we don't do it anywhere else..

Reply:
cookicutter-module PR?